lxd/daemon.go

```diff
@@ -395,30 +394,28 @@ func (d *Daemon) Init() error {
 		return err
 	}
 
-	if !d.os.MockMode {
-		/* Read the storage pools */
-		err = SetupStorageDriver(d.State(), false)
-		if err != nil {
-			return err
-		}
+	/* Read the storage pools */
+	err = SetupStorageDriver(d.State(), false)
+	if err != nil {
+		return err
+	}
 
-		/* Apply all patches */
-		err = patchesApplyAll(d)
-		if err != nil {
-			return err
-		}
+	/* Apply all patches */
+	err = patchesApplyAll(d)
+	if err != nil {
+		return err
+	}
 
-		/* Setup the networks */
-		err = networkStartup(d.State())
-		if err != nil {
-			return err
-		}
+	/* Setup the networks */
+	err = networkStartup(d.State()) // No-op if MockMode is on.
+	if err != nil {
+		return err
+	}
 
-		/* Restore simplestreams cache */
-		err = imageLoadStreamCache(d)
-		if err != nil {
-			return err
-		}
+	/* Restore simplestreams cache */
+	err = imageLoadStreamCache(d) // No-op if no simplestreams.yaml metadata file exists.
```

Comment:
Same here

Reply:
Fixed in 0d62d4fbef6355d16f935fe0f84efdee97a61a20 as well.